src/modules/article/__tests__/articleSelectors-test.js

```diff
@@ -51,7 +47,7 @@ test('articleSelectors getArticle (nb locale)', () => {
 });
 
 test('articleSelectors getArticle (en locale)', () => {
-  const getArticleSelector = getArticle('1');
+  const getArticleSelector = getArticle('4');
```

Comment:
Hvorfor har du endret fra 1 til 4?

Reply:
Fordi man ikke får tilbake title array så måtte legge til en artikke til, da nummer 4 og denne har bare tittel på engelsk

Reply:
Da mister vi jo test for tags, created, updated, metaDescription, content, visualElement og introduction.